### DISCUSSION TITLE
Style applied to wrong line

### DISCUSSION THREAD

**Discussion by PatrickRogg** [outline/rich-markdown-editor#228] [General]
Hello,

first of all thanks for creating such a nice editor. I noticed some weird behaviour. I add a letter to a line, go to the next line and this line is also the last line in the editor and then apply a command the styling is applied to the prev line. I made a short video about the issue: [click](https://streamable.com/s8qjdm).

If the line is not the last line the style is applied to the correct line. Also on outline this does not happen. How can I prevent this from happening?

Thanks in advance!

**Comment by tommoor** [comment]
I think it would be fair to file this as a bug, although definitely strange if you're unable to reproduce in Outline – it is using the latest version of the editor right now.
